desktop/cmp/tab/TabSwitcher.js

```diff
@@ -27,6 +27,9 @@ export class TabSwitcher extends Component {
     static modelClass = TabContainerModel;
 
     static propTypes = {
+        /** Primary component model instance. */
+        model: PT.oneOfType([PT.instanceOf(TabContainerModel), PT.object]).isRequired,
```

Comment:
Should only include the model class, not an object.  This component is a helper comp that works alongside a TabContainerModel by interacting with its model - it would never instantiate such a model on its own.